src/Model/GiftCard.php

```diff
@@ -49,6 +49,8 @@ class GiftCard implements GiftCardInterface
 
     protected ?DateTimeInterface $expiresAt = null;
 
+    protected bool $sendNotificationEmail = true;
```

Comment:
I think this should be false by default. See my comment here: https://github.com/Setono/SyliusGiftCardPlugin/pull/219#issuecomment-1119427019

Reply:
No, he would not since the email is on gc_create event

Reply:
Okay. Great. Thanks